spec/requests/api/v3/user/user_resource_spec.rb

```diff
@@ -64,19 +64,19 @@
       it 'contains the user in the response' do
         expect(subject.body)
           .to be_json_eql(current_user.name.to_json)
-          .at_path('_embedded/elements/0/name')
+                .at_path('_embedded/elements/0/name')
```

Comment:
Why the extra indentation here (and all the other `.at_path` cases)? I think it's correct the way it was before unless you want to increase the indentation with every extra cal in the chain.

If a tool did the formatting (presumably like all the hash spacing corrections) the tool is wrong IMHO.

Reply:
This is due to Rubymine not correctly parsing these rspec blocks. There's no way to configure this properly, but they get aligned with the argument of `.to` since that's a function call